modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/src/Entity/WebtoolsAnalyticsRule.php

```diff
@@ -78,4 +86,12 @@ public function getRegex(): string {
     return $this->regex;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function isSupportMultilingualAliases(): bool {
+    $moduleHandler = \Drupal::service('module_handler');
```

Comment:
This should be moved away from here and renamed to something like `appliesToDefaultLanguageAlias()` if we follow my idea on the first comment above.
At that point, it only returns the configured value.

Reply:
This now should become:

```php
  public function matchOnSiteDefaultLanguage(): bool {
  ...
  }
```

Reply:
Done